Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.yml

```diff
@@ -581,7 +581,7 @@ script:
     - contextPath: Splunk.UserMapping.SplunkUser
       description: Splunk user mapping.
       type: String
-  dockerimage: demisto/splunksdk:1.0.0.33029
+  dockerimage: devdemisto/splunksdk-py3:1.0.0.37176
```

Comment:
Why is the image from `devdemisto` and not `demisto`?

Reply:
will change once it's ready

Reply:
now is the time